packages/@okta/vuepress-site/docs/guides/deploy-to-production/java/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Java
+---
+
+Java applications typically build into a WAR or a JAR for production.
+
+If you deploy your application as a WAR, it's possible you have a context path. If you do, make sure you add this to your Login redirect URI and your Logout redirect URI for your Okta app.
+
+<!--
+// todo: show how to do this with the Okta CLI
+-->
+
+JAR-based Java apps usually don't have a context, and if you start them locally, they'll be available at `http://localhost:8080`.
```

Comment:
avoid future tense (which also helps to avoid "be" verbs):

JAR-based Java apps usually don't have a context, and if you start them locally, they are available at `http://localhost:8080`.